frontend/src/metabase/common/components/FilterPicker/FilterColumnPicker/FilterColumnPicker.tsx

```diff
@@ -0,0 +1,143 @@
+import { useMemo } from "react";
+import { t } from "ttag";
+
+import { getColumnIcon } from "metabase/common/utils/columns";
+import {
+  getColumnGroupIcon,
+  getColumnGroupName,
+} from "metabase/common/utils/column-groups";
+
+import { Icon } from "metabase/core/components/Icon";
+import type { IconName } from "metabase/core/components/Icon";
+
+import * as Lib from "metabase-lib";
+
+import { StyledAccordionList } from "../FilterPicker.styled";
```

Comment:
Can we please move this to `FilterColumnPicker.styled`? Other components won't need the `StyledAccordionList`